modules/data.atmosphere/R/check_met_input.R

```diff
@@ -45,20 +45,20 @@ check_met_input_file <- function(metfile,
     "T[[:digit:]]{2}:[[:digit:]]{2}:[[:digit:]]{2}Z$"
   )
   test_dims <- list(
-    try2(testthat::expect_type(dimensions, "list")),
-    try2(testthat::expect_equal(length(dimensions), 3)),
-    try2(testthat::expect_true("time" %in% names(dimensions))),
-    try2(testthat::expect_match(
-      ncdf4::ncatt_get(nc, "time", "units")[["value"]],
-      time_regex
+    try2(assertthat::assert_that(is.list(dimensions))),
+    try2(assertthat::are_equal(length(dimensions), 3)),
+    try2(assertthat::assert_that("time" %in% names(dimensions))),
+    try2(assertthat::assert_that(str_match(
```

Comment:
build is throwing an error because str_match is undefined. Needs namespace

Reply:
```suggestion
    try2(assertthat::assert_that(stringr::str_match(
```